.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: csharp
-dotnet: 2.1.4
+dotnet: 2.1.301
```

Comment:
Version 2.1.300 had an introduction of node re-use which caused a vstest issue that was fixed in 2.1.301. Using this version fixes some other dotnet test behaviors.